app/views/pages/resources.html.erb

```diff
@@ -98,5 +98,22 @@
         </ul>
       </div>
     </div>
+
+    <div class="usa-grid">
+      <div class="usa-width-one-third">
+        <h4>Forms</h4>
+      </div>
+      <div class="usa-width-two-thirds">
+        <ul class="usa-unstyled-list">
+          <li><%= link_to 'DD1791 - Counseling Checklist', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd1797.pdf', class: "usa-external_link" %>: This form will be automatically generated by the Defense Personal Property system upon completion of self counseling.</li>
+          <li><%= link_to 'DD1299 - Application for Shipment and Storage', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd1299.pdf', class: "usa-external_link" %>: This form will be automatically generated by the Defense Personal Property system upon completion of self counseling when you indicate storage is requested.</li>
+          <li><%= link_to 'DD2278 - Application for PPM (DITY) and Counseling Checklist', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd2278.pdf', class: "usa-external_link" %>: This form will be automatically generated by the Defense Personal Property system upon completion of self counseling for a PPM.</li>
+          <li><%= link_to 'SF1038 - Advance of Funds Application (PPM)', 'https://app_gsagov_prod_rdcgwaajp7wr.s3.amazonaws.com/SF1038-77.pdf', class: "usa-external_link" %>: This form needs to be completed and submitted to your local finance office if you will be requesting an advance of funds to help cover some of the expenses associated with your move.</li>
+          <li><%= link_to 'DD1351-2 - Travel Voucher', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd1351-2.pdf', class: "usa-external_link" %>: This form needs to be completed and submitted to your local finance office at the completion of your move so you can claim all your reimbursable expenses that you are entitled to during your move. Some finance offices have online systems for this form in place, however it’s listed here if needed.</li>
+          <li><%= link_to 'Sample Inconvenience Claim Letter', asset_path('claim.pdf'), download: 'claim.pdf' %>: Included is a sample inconvenience claim letter to use as an example. For more information on an Inconvenience claim, please review this <%= link_to 'Frequently Asked Question', faqs_path %>.</li>
+          <li><%= link_to 'PPM Submission Checklist', asset_path('ppm_checklist.pdf'), download: 'ppm_checklist.pdf' %>: The Personally Procured Move checklist will help you gather all the required documents prior to submitting your PPM voucher to the personal property office.</li>
```

Comment:
I used this to make the last 2 forms downloadable from assets. I saw some other methods for doing this on stack overflow that used a controller. Couldn't figure out why though if this works.

Reply:
The upside of placing these PDF assets in `app/assets/documents` is that they'll be treated similarly to images, stylesheets, etc. by the Asset Pipeline. When running, the application will assign them unique hashes for their file names for cache busting (a good thing!).

Alternatively, these files could be placed in `public/documents` (or similar) and be referenced like:

```erb
<%= link_to 'Sample Inconvenience Claim Letter', '/documents/claim.pdf', download: 'claim.pdf' %>
```

In this case, the Asset Pipeline isn't involved so there's lower application overhead (a good thing!) but the trade off is you don't get the hashed file names. You'd instead have to take into consideration the Web server's caching rules  for PDFs.

All that's to say… given our setup here, placing these files in `app/assets/documents` is the wiser choice. 👍